src/transformers/__init__.py

```diff
@@ -888,6 +892,22 @@
             "load_tf_weights_in_bert",
         ]
     )
+    _import_structure["models.ernie"].extend(
+        [
+            "ERNIE_PRETRAINED_MODEL_ARCHIVE_LIST",
+            "ErnieForMaskedLM",
+            "ErnieForMultipleChoice",
+            "ErnieForNextSentencePrediction",
+            "ErnieForPreTraining",
+            "ErnieForQuestionAnswering",
+            "ErnieForSequenceClassification",
+            "ErnieForTokenClassification",
+            "ErnieLayer",
```

Comment:
```suggestion
```
This one should not be imported here. It will be accessible via the submodule.

Reply:
Yes BertLayer shouldn't have been exposed as well, but we won't remove it now for BC reasons.